src/palace/manager/data_layer/identifier.py

```diff
@@ -27,7 +28,14 @@ def from_identifier(cls, identifier: Identifier | IdentifierData) -> Self:
         return cls(type=identifier.type, identifier=identifier.identifier)
 
     def redis_key(self) -> str:
+        """
+        String representation of the IdentifierData object suitable for use
+        as a redis key.
+        """
         return (
             f"{self.__class__.__name__}{RedisKeyGenerator.SEPERATOR}"
             f"{self.type}{RedisKeyGenerator.SEPERATOR}{self.identifier}"
         )
+
+    def __str__(self) -> str:
+        return f"{self.type}/{self.identifier}"
```

Comment:
All the changes in this file are not necessary for this PR. I can pull them out of desired. They were just helpful to me when debugging, and I thought they were changes worth bringing into the codebase.